sessions/js-forms/demo-start/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML Forms</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+		<script src="./js/index.js" defer></script>
+	</head>
+	<body>
+		<main>
+			<h1>HTML Form Session</h1>
+			<h2>Email Subscription Form</h2>
+			<p>
+				Please fill out this form if you want to receive updates you don't want to read,
+				ever.
+			</p>
+			<form data-js="form">
+				<fieldset>
+					<legend>Personal Information</legend>
+					<label for="first-name">First Name</label>
+					<input
+						type="text"
+						name="firstName"
+						id="first-name"
```

Comment:
I can already foresee a possible question from a student: "Why do we mix camel/kebap-case, when do we use what and why"?
Could we name the form field´s attributes `name` and `id` the same?

Reply:
Yes - absolutely! If I remember @CJAkkuec correctly, this was meant as an explicit hook to stumble upon in explanation of the form.

The students should see and understand, that it's not advisable to use `name="first-name"`. Therefore this little example was added to the demo.

I think we could change this in the demo. I have no strong opinion about this. 

It's easier to understand for students, but skips the discussion of this topic.

Reply:
Just to clarify: why would you say "it's not advisable to use `name="first-name"`" – because of the possible receiving side / the server, which would prefer the field name in camelCase notation?

Reply:
I my opinion it's not advisable to use kebab-case because students need to understrand the dot notation doesn't work to access the value. 

```js
form.addEventListener('submit', event => {
  event.preventDefault();

  // syntax error :-/
  console.log(
    event.target.elements.first-name.value
  );
});
```

Using camelCase for  the `name` - Attribute rules out this issue.

Reply:
I changed `id` and `for` attributes to camelCase as you suggested in this discussion, see the commit b7f1f49307140314b62c828ebab8f4491edd2765.